docs/player-support/squeezelite.md

```diff
@@ -1,18 +1,28 @@
 
 ## Features
 
-- Squeezelite devices are auto detected in Music Assistant
-- Squeezelite devices will play in sync
-- Any physical control buttons on the device should be supported as long as [flow mode](../faq/tech-info.md/#track-queueing) is not enabled
-- The player settings allow configuration of stereo pairs of speakers
+- Squeezelite client devices are automatically detected in Music Assistant.
+- Squeezelite client devices will play in perfect sync and support sync correction.
+  - Speaker sync group support (i.e. support for permanent grouping players).
+- Squeezelite client devices have support for crossfade and gapless playback.
+- Player settings allow the configuration of stereo pairs of speakers.
+- Individual player DSP settings will be used for playback.
+- Up to 384kHz max at 24bit sample rate for PCM if supported by the client device.
+- Lossless Stream support.
+- Squeezelite client device buttons support.
+  - Any physical control buttons on the device should be supported as long as [flow mode](../faq/tech-info.md/#track-queueing) is not enabled.
 
 ## Known Issues / Notes
 
-- The focus of this provider in MA is to support the use of software Squeezelite clients (e.g. [piCorePlayer](https://www.picoreplayer.org/)). Old Squeezebox hardware is reported to work although there are reported issues with grouping. The MA core team cannot afford to expend resources on making these devices work any better than they do now. PRs to enhance support, however, are welcomed to the [aioslimproto library](https://github.com/home-assistant-libs/aioslimproto) and/or the squeezelite provider in MA
-- Running LMS on the same server as MA with the MA Squeezelite provider enabled can have undesired effects
-- Make sure that you do not have the "slimproto" integration running in Home Assistant
-- Ordinarily do not use queue flow mode unless using older Squeezebox hardware player that has issues transitioning. Enabling queue flow mode may solve playback issues however it might come with the side effect of disabling actual physical buttons and/or display of metadata on the device itself
+- The focus of this provider in MA is to support the use of software Squeezelite clients (e.g. [piCorePlayer](https://www.picoreplayer.org/)). Old original Squeezebox hardware is reported to work although there are reported issues with grouping. The MA core team cannot afford to expend resources on making these devices work any better than they do now. PRs to enhance support, however, are welcomed to the [aioslimproto library](https://github.com/home-assistant-libs/aioslimproto) and/or the squeezelite provider in MA.
+  - Some features can be limited or missing if not fully supported by the Squeezelite client player itself.
```

Comment:
What features are you referring to?

Reply:
With that sentence I am refering to limitations on the Squeezelite client device player side, such as for example if the Squeezelite client device playe does not support 384kHz 24bit then it can play that. So it is not a limitation on the server-side but only the client-side.

> _**Some features can be limited or missing if not fully supported by the Squeezelite client player itself.**_

Reply:
Removed that line now.